### DISCUSSION TITLE
See task volunteer on admin events list

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#34] [General]
Is there a way to see who has volunteered for a task in the Events list in the admin back end? I know that I can go the Tasks Signups page, but I have several repeating events that need a volunteer to run them, and I have to repeatedly compare the Events list with the Tasks Signups list to make sure that every upcoming event has a volunteer. 

It would be easier if the Events list header row had a right-click option to add the Tasks Signups. Is this possible?

**Comment by liedekef** [comment]
Do you mean in the event list overview, together with "RSVP Info"? That would be possible ...

**Reply by TimB111** [reply]
Yes in the events list overview, when you go WordPress > EME > Events.

So when you see the list of events, it would be helpful to add a column called "Task signups" where it would show who has signed up. 

Or even if the column showed the number of tasks and the number of volunteers signed up/approved then it would make it a lot clearer and simpler to see in one place.

**Comment by liedekef** [comment]
There is no approving (yet), but this should help:
https://github.com/liedekef/events-made-easy/commit/dea5b0b9d028846dd7e4fe45e70983b47b1e9f53

**Reply by TimB111** [reply]
Thanks, I updated just this file eme-rsvp.php but it doesn't show anything different on the events list page. Trying to read the code (as a non-coder) it looks like it should show the number of tasks and the number of volunteers below the event name in the list, but this doesn't appear.

**Reply by liedekef** [reply]
Well, the code I gave was to show the Task info, but on the Bookings page (and it contained an error). This should help:
https://github.com/liedekef/events-made-easy/commit/3ddce29f2484db35c50363a72d626c646dd0cbfd

**Reply by TimB111** [reply]
Great, it shows the task used/total count on the Bookings page.

I've also updated the eme-events.php file but this doesn't show the task count on the Events list page.

**Comment by liedekef** [comment]
Indeed, small typo. This is the (hopefully) final fix for that: https://github.com/liedekef/events-made-easy/commit/e30bb6c095a79c683f0d46875fd870e4b31ff5ed (just a small fix in eme_events.php, you can ignore the other change which is just a tab-correct)

**Reply by TimB111** [reply]
This works now, thank you!
